balance/doc.go

```diff
@@ -77,3 +77,17 @@ when NeoFS contract has transferred GAS assets back to the user.
 	    type: Integer
 */
 package balance
+
+/*
+Contract storage model.
+
+# Summary
+Key-value storage format:
+ - 'MainnetGAS' -> int
+   total amount of Mainchain GAS deployed in the NeoFS network
```

Comment:
In Fixed12? Looks like. And it's important since mainnet GAS is Fixed8.

Reply:
how constant is this precision overall? Contract is NEP-17 so it provides `Decimals` method

maybe it's clearer to write `with precision determined by the current contract`? Exact precision is described in contract docs and, in theory, can be changed. In storage we have generic `int` anyway

Reply:
> how constant is this precision overall?

It can't change for a deployed NEP-17. So I doubt it can be changed in our contracts. Not until we change the internal token to something else.

> maybe it's clearer to write with precision determined by the current contract

The main problem is that we're talking main chain GAS here and it's known to have precision of 8, so one can expect 1 GAS stored as 100000000, while in fact that's not the case, it's an internal representation of this GAS which currently has precision of 12 (and as I've said _very_ unlikely to change).

Reply:
got ur point

clarified